TokboxSessionHelpers.js

```diff
@@ -0,0 +1,126 @@
+import debug from 'debug'
+import { isAttendee, showConnectionLostMessage } from 'libs/helpers'
+import { emitLocalEvent, onLocalEvent } from 'libs/localEvents'
+import { autoplayStream, destroyAutoplayStream } from 'libs/audioAutoplay'
+import { tokboxSubscribe } from 'libs/TokboxSubscriber'
+import { Kit } from 'demio-ui-kit/src'
+
+const { notification } = Kit
+const log = debug('demio:TokboxSessionHelpers')
+const { user: { role } } = window.pageData    
+const isUserAttendee = isAttendee(role)
+
+// Session
+
+export function sessionReconnecting () {
```

Comment:
I would change the error message. The `sessionReconnecting` event is triggered when the JS library detects that the client has disconnected (https://tokbox.com/developer/guides/connect-session/js/#automatic_reconnection)

So the message could be something like: "You have lost connection. Please wait a few seconds as we are trying to reconnect you to the session"